makefile

```diff
@@ -182,3 +183,6 @@ style:
 	@if ! $(ASTYLE) $(FORMAT_FILES) | grep Formatted; then\
 	   echo "No source files were changed.";\
 	fi
+
+$(shell mkdir -p run)
+$(shell mkdir -p run/ROMsol)
```

Comment:
It would also be convenient to have `make clean` or `make cleanrun` do `rm -rf run`. This seems necessary when doing multiple runs, since the files from two runs can get mixed up.

Reply:
added!